src/app/leaves/add/components/LeaveRequestFormBuilder.tsx

```diff
@@ -0,0 +1,262 @@
+"use client";
+import React from "react";
+import FormWrapper from "@/common/FormWrapper";
+import { cn } from "@/lib/utils";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { CalendarIcon } from "@radix-ui/react-icons";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { format } from "date-fns";
+import { Button } from "@/components/ui/button";
+import { Calendar } from "@/components/ui/calendar";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { Textarea } from "@/components/ui/textarea";
+import { MembersProps } from "@/types";
+import { LeaveTypes } from "@/constant/constant";
+import { MemberSelector } from "@/common/MemberSelector/MemberSelector";
+
+export const leaveRequestSchema = z
+  .object({
+    memberId: z.string().min(1, {
+      message: "Member must be selected.",
+    }),
+    leaveType: z.string().min(1, {
+      message: "Leave Type must be selected.",
+    }),
+    startDate: z
+      .date({
+        message: "Start Date must be selected.",
+      })
+      .refine((date) => date >= new Date(), {
+        message:
+          "Start date must be greater than or equal to the current date.",
+      }),
+    endDate: z
+      .date({
+        message: "End Date must be selected.",
+      })
+      .refine((date) => date >= new Date(), {
```

Comment:
can we remove = sign form here  here

Reply:
No, because the user will be able to select the current date.